sdk/samples/iot/templates/iothub/test-resources.json

```diff
@@ -0,0 +1,87 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "hubname": {
+            "type": "string",
+		"defaultValue": "aziotbld-embed-cd"
+        },
+        "location": {
+            "type": "string",
+		"defaultValue": "eastus"
```

Comment:
Should this be in US East?

Reply:
looks like the default for the resource group is westus2, will change.